python2.7/run/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN rm /var/runtime/awslambda/runtime.so
 ADD runtime-mock.py /var/runtime/awslambda/runtime.py
 
 # Not sure why permissions don't work just by modifying the owner
-RUN rm -rf /tmp && mkdir /tmp && chown -R sbx_user1051:495 /tmp && chmod 700 /tmp
+RUN rm -rf /tmp && mkdir /tmp && chown -R sbx_user1051:495 /tmp && chmod 700 /tmp && chmod -R 0777 /var/runtime
```

Comment:
Mmh probably not, you're right. Especially since I guess the normal use case will be to run it as a one-off anyway.